opencodelists/forms.py

```diff
@@ -96,7 +96,9 @@ def validate_csv_data_codes(coding_system, codes):
     if not coding_system.has_database:
         return
     unknown_codes = set(codes) - set(coding_system.lookup_names(codes))
-    unknown_codes_and_ixs = [(codes.index(code), code) for code in unknown_codes]
+    unknown_codes_and_ixs = sorted(
+        [(codes.index(code), code) for code in unknown_codes]
+    )
```

Comment:
Does this sort on the first element of the tuple?

Reply:
yes; initally I put a key, but it doesn't need one

Reply:
Ah, but it will be faster if it uses a key, given we don't care about the second element.  I will add one :)